edc-extensions/bdrs-client/src/main/java/org/eclipse/tractusx/edc/identity/mapper/BdrsClientImpl.java

```diff
@@ -120,7 +120,7 @@ private boolean isCacheExpired() {
     private Result<Void> updateCache() {
         var membershipCredToken = createMembershipPresentation();
         if (membershipCredToken.failed()) {
-            return membershipCredToken.mapTo();
+            return membershipCredToken.mapEmpty();
```

Comment:
Any reason to use `mapEmpty` instead of `mapFailure`? I'd personally prefer the latter since IMO it's more indicative of a failure.

Reply:
No reason :)